common/argo/base/deployment.yaml

```diff
@@ -37,7 +37,7 @@ spec:
           value: "false"
         - name: BASE_HREF
           value: /
-        image: argoproj/argoui:v2.4.2
+        image: argoproj/argoui:v2.5.0-rc8
```

Comment:
@rushtehrani This isn't going to work I believe.
You need to change these values inside `kustomization.yaml`

Reply:
@aleksandrmelnikov I pointed to this branch and deployed via CLI and it seemed to work fine, did you try it?

Reply:
Hmm, let me test. I ran into this issue earlier with workflow-controller